changelog/57529.fixed

```diff
@@ -0,0 +1,2 @@
+Set ``enable_fqdns_grains`` to default to ``False`` on Windows. This addresses
+some of the slowness we're seeing.
```

Comment:
The changelog entries are meant for end users. This file should be renamed to `57529.changed`, since this is more of a change in behavior, not really fixing incorrect behavior (though, speed *is* a feature). Or perhaps `.added`, since we're adding a config option.



Also, the text could be more end-user friendly:

> Added `enable_fqdns_grains` config option to address some slowness on Windows.

Reply:
Dun